docs/notebooks/util/plotting.py

```diff
@@ -16,10 +16,11 @@
 import numpy as np
 from matplotlib import cm
 
+from trieste.type import TensorType
 from trieste.utils import to_numpy
 
 
-def create_grid(mins, maxs, grid_density=20):
+def create_grid(mins: TensorType, maxs: TensorType, grid_density=20):
```

Comment:
if this can only be a 2 element TensorType, then maybe you should assert that below.